draft-ietf-openpgp-pqc.md

```diff
@@ -893,6 +896,9 @@ algorithm.
 
     MB = KMAC256(domSeparation, encData, oBits, customizationString)
 
+Here, the inputs to KMAC256 have the meaning key, main input data, requested
```

Comment:
> It should be made more clear that this is meant as a mapping between the KMAC terminology `KMAC256(K, X, L, S)` and the inputs that we pass in the combiner. I think explicitly naming the parameters `K, X, L, S` and referencing to Section 4.2 and 4.3 in SP800-185 will make it more clear.

OK, fixed.
> 
> Implementers will face the additional challenge of mapping these values to the MAC API of their chosen crypto library which can be unintuitive. For example, the nonce/iv will be the `customizationString`. It's probably out of scope for this document, but I'd not be surprised if this confuses some people. So my question is: Can or should we add some hints in the document?

I don't see that we can clarify the use of library interfaces for implementers on that level. Then we would have do that for each and every cryptographic operation that we mention. Each time this would involve lengthy text full of speculation how a library might have to be used. I think this would do more harm than good in terms of clarity.